### PR TITLE
name convention bugfix

### PR DESCRIPTION
**Pull request by triccyx**
Some name conventions fix.

### CODE REVIEW

File: plugins/actuator/README.md [hunk 1]
```diff
@@ -99,6 +99,14 @@ A common language used for generating actuator or trasmission models in the form
 an object-oriented, equation based language to conveniently model complex physical systems. Modelica is used in several  Simulation Environments, such as [OpenModelica](https://openmodelica.org/) or [Dymola](https://www.3ds.com/products-services/catia/products/dymola).
 Please see the [`doc/modelica-actuator.md`](doc/modelica-actuator.md) document for additional insight on how to use the gazebo-fmi Actuator plugin with Modelica models.
 
+## Naming conventions
+
+In this software the following main name conventions are used:
+* Type names start with upper case.
+* Class member variabile names start with m_ then lower case.
+* Class member method names start with upper case.
+* Local variabile names start with lower case.
```
[the remark this answers is not in the document]
Comment (triccyx) [reply plugins/actuator/README.md]: Agree.
However, do you think it is a good idea to add a link to the new outcoming file Contributing.md 
in the readme.md?
Something like

>  If you want to contribute check ...

Is it possible to define a Contributing.md for all our lab projects?

[the remark this answers is not in the document]
Comment (traversaro) [reply plugins/actuator/README.md]: > However, do you think it is a good idea to add a link to the new outcoming file Contributing.md
in the readme.md?

Yes, it make sense. 

> Is it possible to define a Contributing.md for all our lab projects?

Good luck getting everyone to agree on the content of the file. : ) 
I think it is easier to just add in this file what we need in this project, and then link https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md for further details.

[the remark this answers is not in the document]
Comment (triccyx) [reply plugins/actuator/README.md]: > Good luck getting everyone to agree on the content of the file. : )

A very common problem!

[the remark this answers is not in the document]
Comment (triccyx) [reply plugins/actuator/README.md]: I like and use (when possibile):

> https://google.github.io/styleguide/cppguide.html

[the remark this answers is not in the document]
Comment (traversaro) [reply plugins/actuator/README.md]: A rough consensus on a clang-format style similar to what is used in YARP was kind of reached in https://github.com/robotology-playground/clang-format/issues/1 , it is the one that we used in https://github.com/robotology/sdf-modelica/pull/21 .

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+----------
+A set of conventions about how to write code for this project.
+
+## Naming conventions
+* Type names start with upper case.
+* Class member variabile names start with m_ then lower case.
+* Class member method names start with upper case.
+* Local variabile names start with lower case.
+
+For that not foreseen in the current document refer to:
```
Comment (traversaro) [inline Contributing.md]:
```suggestion
For any additional detail not covered by this document, please refer to
```

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Contributing to gazebo-fmi
+==========================
+
+Code Style
+----------
+A set of conventions about how to write code for this project.
+
+## Naming conventions
+* Type names start with upper case.
+* Class member variabile names start with m_ then lower case.
```
Comment (traversaro) [inline Contributing.md]:
```suggestion
* Class member names start with `m_` followed by a lower case.
```

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+* Class member method names start with upper case.
+* Local variabile names start with lower case.
```
Comment (traversaro) [inline Contributing.md]:
```suggestion
* Local variable names start with lower case.
```

**Comment by traversaro** [comment]
Thanks!

**Pull request merged**